pkg/server/commands/write_authzmodel.go

```diff
@@ -38,11 +39,18 @@ func WithWriteAuthModelMaxSizeInBytes(size int) WriteAuthModelOption {
 	}
 }
 
+func WithWriteAuthModelEnableModularModels(enable bool) WriteAuthModelOption {
```

Comment:
```suggestion
func WithEnableModularModels(enable bool) WriteAuthModelOption {
```

Reply:
I agree this name is stupendously long, but it's in keeping with the existing `WithWriteAuthModelLogger` and `WithWriteAuthModelMaxSizeInBytes`, as well all other options throughout this folder follow the same `With<Command><Option>` pattern.

I'm assuming they are all intentionally prefixed to help differentiate usage as they're all under the `commands` namespace

Reply:
nvm, I didn't refresh the page this morning and hadn't noticed this change was made